ts/Core/Legend/Legend.ts

```diff
@@ -1398,7 +1398,7 @@ class Legend {
             // PDF export (#1787)
             if (!clipRect) {
                 clipRect = legend.clipRect =
-                    renderer.clipRect(0, padding, 9999, 0);
+                    renderer.clipRect(0, padding - 2, 9999, 0);
                 legend.contentGroup.clip(clipRect);
```

Comment:
This creates the clipping rectangle **only** when the legend has too many elements and the navigation is added. It is not created for a legend with just a few elements.

The `x, width, height` for this `clipRect` are set correctly, but `y` is not. From what I deduced from the code `y` is not calculated anywhere, `clipRect.y` is simply `padding` and it's only set in the `1401` line, nowhere else. So reducing it by a few pixels here solves the problem.

I know it's still hardcoding a value, but I believe it's a better solution than the previous one because we only affect the clipping rectangle which is only created when there is `legend.nav`

Reply:
Yes I see. I think the two pixels account for the symbol (circle) extending above the text. So a perfect solution would be to subtract the actual symbol height from the line height. But I think this is good enough.